scripts/test/shared.py

```diff
@@ -224,15 +224,13 @@ def has_shell_timeout():
 
 
 # Default options to pass to v8. These enable all features.
+# See https://github.com/v8/v8/blob/master/src/wasm/wasm-feature-flags.h#L73-L82
```

Comment:
This range is only about shipped features. I wonder other info (staged and non-staged features) are relevant too? And also the line numbers are subject to change, unless we take a permalink for it. How about just including the link of a file itself, without line numbers? The whole file is relevant info I think.

Reply:
Sounds good.